internal/msalbase/auth_parameters_internal.go

```diff
@@ -35,8 +35,10 @@ type AuthParametersInternal struct {
 }
 
 //CreateAuthParametersInternal creates an authorization parameters object
```

Comment:
space before, period after + potentially rename to NewAuthParametersInternal

Reply:
... Probably a lot of renames if we move away from Create*, this isn't the place for that.

Reply:
Updated.  Going to hold renames for a future change.